sys/include/phydat.h

```diff
@@ -79,6 +79,8 @@ enum {
     UNIT_TEMP_C,    /**< degree Celsius */
     UNIT_TEMP_F,    /**< degree Fahrenheit */
     UNIT_TEMP_K,    /**< Kelvin */
+    /* luminosity */
```

Comment:
Maybe just "electromagnetism" and merge it with "electricity" so that we are definitely on the safe side. Just kidding, I'm fine with luminosity, since units like e. g. cd or lm then can find their way here two.